src/main/cpp/subsystems/Elevator.cpp

```diff
@@ -157,3 +167,22 @@ double Elevator::GetLeftVoltage() {
 bool Elevator::IsRevLimitSwitchClosed(){
   return mLeftElevator.GetSensorCollection().IsRevLimitSwitchClosed();
 }
+
+double Elevator::EncoderTicksToInches(double inches){
+  int encoder1 = 0;
+  int encoder2 = 13000;
+  int inches1 = 9;
+  int inches2 = 27;
```

Comment:
These hardcoded values should not be hardcoded. 

The bottom and top inches can be hard coded. These are _NOT_ the right values.